bin/tessel-2.js

```diff
@@ -287,18 +287,27 @@ module.exports.closeSuccessfulCommand = function() {
 };
 
 // Allow options to be partially applied
-module.exports.closeFailedCommand = function(opts, err) {
-  if (!err) {
-    err = opts;
-    opts = {};
-  }
-  if (err instanceof Error) {
-    throw err;
+module.exports.closeFailedCommand = function(opts, error) {
+
+  if (opts) {
```

Comment:
I found no occurrences where `closeFailedCommand` is called with nothing, so I'm re-working this. I was also able to clean up the syntax error handling from https://github.com/tessel/t2-cli/commit/0726002569bb3d043e0b22edb2a5ac51e86b304c